src/Clock.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Clock extends React.Component {
+  state = {
+    time: new Date().toLocaleTimeString(),
+  };
+
+  componentDidMount() {
+    this.startClock();
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (prevState.name !== this.props.name) {
+      window.console.log(
+        // eslint-disable-next-line
+        `The Clock was renamed from ${prevState.name} to ${this.props.clockName}.`,
+      );
+    }
```

Comment:
```suggestion
  componentDidUpdate(prevProps) {
    if (prevProps.name !== this.props.name) {
      window.console.log(
        // eslint-disable-next-line
        `The Clock was renamed from ${prevProps.name} to ${this.props. name}.`,
      );
    }
```
Be careful with namings. Prop is called `name`, not `clockName`
Your app isn't working properly at the moment:
https://www.loom.com/share/479c0854b9d7412ea61b3ff40b82d668

The phrase `The Clock was renamed from oldName to newName.` should only be logged into the console when clicking on the button 'Set random name' (not every second)